ansible/roles/miniflux/defaults/main.yml

```diff
@@ -177,7 +169,7 @@ miniflux__upstream_platform: 'linux-amd64'
 #
 # The URL of the upstream :command:`git` repository which contains
 # Miniflux source code.
-miniflux__upstream_git_repository: 'https://github.com/miniflux/miniflux'
+miniflux__upstream_git_repository: 'https://github.com/miniflux/v2'
```

Comment:
I don't know why previous code includes `miniflux/miniflux`. It could be a mistake on my end, but current repository name is `v2`.